docs/setup-guide/run-e2e-test-locally.md

```diff
@@ -0,0 +1,64 @@
+# How to run e2e test in local dev environment
+
+This doc is to describe how to run e2e test in your local dev environment. Please note, local dev environment is not fitting for many existing e2e test cases. So if those tests failed, it's not a surprise.
+
+1 Install kubetest
+
+E2e test is launched by kubetest. Please visit [kubetest](https://github.com/kubernetes/test-infra/blob/master/kubetest/README.md) to understand it. There is the latest info how to install kubetest if the script in this doc is not working.
+
+(This doc suggests and assumes arktos is installed under $GOPATH/src/k8s.io/)
+
+```bash
+cd $GOPATH/src/k8s.io
+git clone https://github.com/kubernetes/test-infra.git
+cd $GOPATH/src/k8s.io/test-infra/
+GO111MODULE=on go install ./kubetest
+```
+
+kubetest will be installed in $GOPATH/bin/. You may want to add $GOPATH/bin/ to $PATH for easily invoking binaries under the path.
+
+2 	kubetest has a limitation: it only runs under directory of Kubernetes.
+   	You can rename arktos directory to kubernetes.
+   Another way which may be better, is to create a soft link to arktos:
+   
+```bash
+cd $GOPATH/src/k8s.io
+ln -s ./arktos kubernetes
```

Comment:
ln -s does not work for me. Your set up is not our normal setup.

When git clone arktos, it creates folder as ~/go/src/arktos. Normally we don't create arktos under k8s.io.

Reply:
Many k8s tools have assumption on directory topo. We have to follow when using the tools. Initially I also put arktos under src, then from time to time I was experiencing errors.

Reply:
It's ok that there are requirement to run kubetest. But softlink does not work. Your case works because the original kubernetes folder are in the same folder.

Reply:
Removed softlink